pypeit/calibrations.py

```diff
@@ -744,13 +749,17 @@ def get_tilts(self):
 
         return self.wavetilts
 
-    def run_the_steps(self):
+    def run_the_steps(self, quiet=False):
```

Comment:
document what `quiet` does

Reply:
Good catch.  This was left in by mistake.  Removed.